CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
+
 # Release Notes
+## 10.0.12 (Unreleased)
+
```

Comment:
This syncs directly to GitBook. We don't want to put the Unreleased header for the version that is getting released.

```suggestion
```